plugin.yml

```diff
@@ -11,6 +11,8 @@ authors:
 src-namespace-prefix: NhanAZ\Track
 website: https://github.com/NhanAZ
 description: Help staffs track players using commands.
+soft-depend:
```

Comment:
`soft-depend` or `softdepend`?

Reply:
> 

Sofe said `soft-depend`.
https://github.com/nhanaz-pm-pl/Track/pull/7#discussion_r805153373

Reply:
wtf it is `softdepend`
https://doc.pmmp.io/en/rtfd/developer-reference/plugin-manifest.html#softdepend

Reply:
```suggestion
softdepend:
```